content/help/faq/distributed-tracing/control-sampling.md

```diff
@@ -0,0 +1,6 @@
+---
+title: How can I control the sampling rate?
+weight: 40
+---
+
+See [this section](/docs/tasks/telemetry/distributed-tracing/overview/#trace-sampling) for more information on how to set the sampling rate.
```

Comment:
a common question is rate as in probabilistic vs rate/time. The best first-start experience and also from a system load pov is a temporal rate such as 100/second vs a probabilistic one such as 0.01% which is not surge protected. If someone can think of an overview sentence to describe what we mean by rate, we could use it here as not everybody thinks "0.01%" when they ask how to control the sampling rate.

Another way to ask is to say "How can I control the volume of traces?" Then, say that currently istio (via envoy) supports a percentage based sampling strategy which you can configure (xx link here)

Later, we can have a quick thing to mention temporal rate whenever that feature happens upstream

Reply:
Took your suggestion. Hopefully captured the spirit correctly.